### ISSUE TITLE
Instance Local Feed filtering CW posts?

### ISSUE THREAD

**Issue by aneillans** [pixelfed/pixelfed#4373]
It seems the Local Feed on the instance default filters any posts that are CWed? Is this configurable? Can't find any options for this, but seems at odds with the Global feed which does not apparently filter.

**Comment by mitexleo** [comment]
You're the admin of that instance?

**Comment by aneillans** [comment]
Yup - had a hunt through the docs and couldn't see anything that seemed relevant but hoping there is a flag I can set.

**Comment by mitexleo** [comment]
It should filter nsfw posts on Global feed too ....

https://docs.pixelfed.org/technical-documentation/config/#pf_hide_nsfw_on_public_feeds

**Comment by aneillans** [comment]
So I checked my config again, turns out I had:

PF_HIDE_NSFW_ON_PUBLIC_FEEDS="true"

This was filtering local feed, but not the global.
Toggling gave me the expected behaviour on local, but would be worth checking to see why Global isn't filtered :)
